src/components/ProfilePage/Education.jsx

```diff
@@ -95,10 +100,18 @@ class Education extends React.Component {
               <h5>{EDUCATION[value]}</h5>
             </React.Fragment>
           ),
-          empty: <EmptyContent onClick={this.handleOpen}>Add education</EmptyContent>,
+          empty: (
+            <EmptyContent onClick={this.handleOpen}>
+              <FormattedMessage
+                id="profile.education.empty"
+                defaultMessage="Add education"
+                description="instructions when the user doesn't have their level of education set"
+              />
+            </EmptyContent>
+          ),
           static: (
             <React.Fragment>
-              <EditableItemHeader content="Education" />
+              <EditableItemHeader content={intl.formatMessage(messages['profile.education.education'])} />
               <h5>{EDUCATION[value]}</h5>
```

Comment:
Do we need to care about the education values?

Reply:
Oh I missed the entire Constants directory, good catch!